### ISSUE TITLE
1.2.4 causes paper-menu-button menu not to dismiss

### ISSUE THREAD

**Issue by garrettn** [Polymer/polymer#3361]
We have a paper-menu-button component that's structured pretty similarly to the example [on the docs page](https://elements.polymer-project.org/elements/paper-menu-button) (with a paper-menu inside, etc.). After the upgrade to Polymer 1.2.4, the menu no longer dismisses when you select an item in it. I'm pretty sure the Polymer library itself is somehow responsible (pinning it to 1.2.3 makes the problem go away), but since this release had so many changes I'm not sure what might be the cause.

I'm having trouble coming up with a runnable example to demonstrate the issue, but I was wondering if anyone else has seen it.

**Comment by edriggers0810** [comment]
We are seeing a similar issue.  I am going to take your suggestion and revert to 1.2.3 to confirm the problem goes away.

**Comment by cdata** [comment]
This has been frequently reported against specific elements, but this is the first time someone has suggested it is a `v1.2.4` regression. For reference, please see: https://github.com/PolymerElements/paper-dropdown-menu/issues/72

**Comment by garrettn** [comment]
We just figured out that we had some custom logic that was intercepting click events on the menu items and stopping propagation, so the menu never got the event to close. When we allow propagation, the menu closes properly.

The strange thing is that our menus worked at all before 1.2.4. I wonder if we were relying on a bug that was fixed in 1.2.4.

Anyway, this is no longer a problem for us. Should I close the issue?
